<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>voirfacture</class>
 <widget class="QMainWindow" name="voirfacture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">*{
font: 8pt &quot;MV Boli&quot;;

font-size:35px;
color: black;
border-radius:0px;
}</string>
        </property>
        <property name="text">
         <string> Facture  des clients</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCommandLinkButton" name="commandLinkButton_quit">
        <property name="styleSheet">
         <string notr="true">
background-color: rgb(207, 207, 207);
color:black;
font-size:20px;
</string>
        </property>
        <property name="text">
         <string>retour</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="commandLinkButton_supfact">
        <property name="styleSheet">
         <string notr="true">
background-color: rgb(207, 207, 207);
color:black;
font-size:20px;
</string>
        </property>
        <property name="text">
         <string>supprimer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font--size:20px;
background:transparent;
color:black;
font-size:20px;</string>
      </property>
      <property name="text">
       <string>Num client :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_numfact">
        <property name="styleSheet">
         <string notr="true">font--size:20px;
background:transparent;
color:black;
font-size:20px;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="pushButton_recherchfact">
          <property name="styleSheet">
           <string notr="true">
background-color: rgb(207, 207, 207);
color:black;
font-size:20px;
</string>
          </property>
          <property name="text">
           <string>Rechercher </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_showallfact">
          <property name="styleSheet">
           <string notr="true">
background-color: rgb(207, 207, 207);
color:black;
font-size:20px;
</string>
          </property>
          <property name="text">
           <string>voir tous les facture</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTableView" name="tableView_fact">
        <property name="styleSheet">
         <string notr="true">font--size:20px;
background-color: rgb(238, 238, 238);
color:black;
</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTableView" name="tableView_totale">
          <property name="styleSheet">
           <string notr="true">font--size:20px;
background-color: rgb(238, 238, 238);
color:black;
</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
